### ISSUE TITLE
Training scripts

### ISSUE THREAD

**Issue by ncan33** [TencentARC/FreeSplatter#15]
Hi!

I am interested in running an experiment re-training the FreeSplatter model with objects with unfavorable view images (not perfectly centered at [0,0,0]). Particularly, I am interested in seeing if the work done by [UFV-Splatter](https://github.com/yfujimura/UFV-Splatter) (LoRA + fine tuning) is necessary, or if FreeSplatter's architecture can handle unfavorable views if the training data is more representative of real world images.

Are the training scripts public for FreeSplatter? I couldn't find them in the repository, so I have been writing my own training scripts based on the information in sections 3.2 and 3.3 of the [paper](https://arxiv.org/pdf/2412.09573). I am currently implementing the pixel alignment loss, but it's quite time consuming, so I am asking the question here in case the training code is public.

Any thoughts @bluestyle97?

**Comment by bluestyle97** [comment]
Sorry, due to the company's policy, we can not make the training code public.
